src/contexts/models/contextManager.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+const uuid = require('uuid');
+
+const database = require('./database/sequelize/sequelizeConnector'),
+    { ERROR_MESSAGES } = require('../../common/consts');
+
+module.exports = {
+    getContexts,
+    saveContext,
+    deleteContext
+};
+
+async function getContexts() {
+    const contexts = await database.getContexts();
+    if (contexts) {
+        return contexts.map(({ dataValues }) => ({
+            id: dataValues.id,
+            name: dataValues.name
+        }));
+    } else {
+        const error = new Error(ERROR_MESSAGES.NOT_FOUND);
+        error.statusCode = 404;
+        throw error;
+    }
+}
+
+async function saveContext(contextName) {
+    const id = uuid();
+    try {
+        await database.saveContext(id, contextName);
+    } catch (err) {
+        const error = new Error(err.message);
```

Comment:
we don't expect any 400 here, the open API validator needs to verify the correctness of the input data, so no need for try-catch